在Windows环境下，`xcap::Window` 在跨线程调用时报线程安全错误，在Ubuntu/Macos上无此问题

在Windows环境下，`xcap::Window` 在跨线程调用时报线程安全错误，在Ubuntu/Macos上无此问题：

## 环境：

OS: Windows11 x64
Rust: 1.86.0
Xcap: 0.5.1

## 报错信息

```
error[E0277]: `*mut c_void` cannot be sent between threads safely
    --> src\worker.rs:423:39
     |
423  |       let listen_thread = thread::spawn(move || {
     |                           ------------- ^------
     |                           |             |
     |  _________________________|_____________within this `{closure@src\worker.rs:423:39: 423:46}`
     | |                         |
     | |                         required by a bound introduced by this call
424  | |         trace!("into thread");
425  | |         process_analysis_loop(context);
426  | |     });
     | |_____^ `*mut c_void` cannot be sent between threads safely
     |
     = help: within `{closure@src\worker.rs:423:39: 423:46}`, the trait `Send` is not implemented for `*mut c_void`
note: required because it appears within the type `windows::Win32::Foundation::HWND`
    --> C:\Users\atopx\.cargo\registry\src\rsproxy.cn-e3de039b2554c837\windows-0.61.1\src\Windows\Win32\Foundation\mod.rs:5670:12
     |
5670 | pub struct HWND(pub *mut core::ffi::c_void);
     |            ^^^^
note: required because it appears within the type `xcap::platform::impl_window::ImplWindow`
    --> C:\Users\atopx\.cargo\registry\src\rsproxy.cn-e3de039b2554c837\xcap-0.5.1\src\windows\impl_window.rs:38:19
     |
38   | pub(crate) struct ImplWindow {
     |                   ^^^^^^^^^^
note: required because it appears within the type `xcap::Window`
    --> C:\Users\atopx\.cargo\registry\src\rsproxy.cn-e3de039b2554c837\xcap-0.5.1\src\window.rs:6:12
     |
6    | pub struct Window {
note: required by a bound in `std::thread::spawn`
    --> C:\Users\atopx\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library\std\src\thread\mod.rs:726:8
     |
723  | pub fn spawn<F, T>(f: F) -> JoinHandle<T>
     |        ----- required by a bound in this function
...
726  |     F: Send + 'static,
     |        ^^^^ required by this bound in `spawn`
```

--- a/src/windows/impl_window.rs
+++ b/src/windows/impl_window.rs
@@ -39,6 +39,9 @@ pub(crate) struct ImplWindow {
     pub hwnd: HWND,
 }
 
+unsafe impl Send for ImplWindow {}
+unsafe impl Sync for ImplWindow {}
+
 fn is_window_cloaked(hwnd: HWND) -> bool {
     unsafe {
         let mut cloaked = 0u32;

